OpenRA.Mods.Common/OpenRA.Mods.Common.csproj

```diff
@@ -816,22 +818,14 @@
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <Target Name="AfterBuild">
-    <MakeDir Directories="$(SolutionDir)mods/common/"/>
-    <Copy
-      SourceFiles="$(TargetPath)"
-      DestinationFolder="$(SolutionDir)mods/common/"/>
```

Comment:
Those changes don't look related to this PR. Do we want to revert this? (I don't mind it really.)